frontends/infinite-corridor/package.json

```diff
@@ -10,6 +10,7 @@
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
     "@mui/icons-material": "^5.8.4",
+    "@mui/lab": "^5.0.0-alpha.91",
```

Comment:
This is where MUI puts their not-quite-ready APIs.

Still, they advertise it in their docs, and it has 500k+ npm downloads per month, so I am comfortable using it.

We do not **need** it (and ditching it wouldn't be very hard), but it significantly simplifies using MUI's tabs. See https://mui.com/material-ui/react-tabs/#experimental-api